ansible/inventory/inventory

```diff
@@ -1,11 +1,8 @@
 [microshift]
-microshift.example.com ansible_host= # microshift node
+microshift-dev.localhost ansible_host= private_ip=
```

Comment:
.localhost name always resolves to 127.0.0.1. Let's just have "microshift-dev" default.